benchopt/cli/main.py

```diff
@@ -99,7 +101,7 @@ def run(benchmark, solver_names, forced_solvers, dataset_names,
             dataset_names=dataset_names,
             objective_filters=objective_filters,
             max_runs=max_runs, n_repetitions=n_repetitions,
-            timeout=timeout, plot_result=plot, pdb=pdb
+            timeout=timeout, plot_result=plot, html = html, pdb=pdb
```

Comment:
No whitespace around `=` for parameters.
The rational is that if there is no whitespace, this is keyword argument, else, this is variable assignment. this makes the code easier to read.
```suggestion
            timeout=timeout, plot_result=plot, html=html, pdb=pdb
```